src/constants.js

```diff
@@ -226,7 +226,8 @@ export const SELECT_OUTPUTS_TIMEOUT = 1000 * 60;
 /**
  * Wallet service URL
  */
-export const WALLET_SERVICE_BASE_URL  = 'https://wallet-service.testnet.hathor.network/';
+export const WALLET_SERVICE_BASE_URL  = 'https://wallet-service.hathor.network/';
```

Comment:
Shouldn't we use `wallet-service.mainnet.hathor.network`? I'm not sure what strategy we've been using.

Reply:
I think this is a valid discussion, @luislhl probably followed the explorer URLs pattern:

https://explorer.hathor.network/ for `mainnet`
https://explorer.testnet.hathor.network/ for `testnet`

I like the current URL, without `.mainnet` but we can discuss this in another issue. What do you think?